src/wblib/figures/hifs.py

```diff
@@ -17,10 +17,12 @@ def get_latest_forecast_issue_time(briefing_time: pd.Timestamp):
     return issued_time
 
 
-def get_dates_of_five_previous_initializations(
+def get_dates_of_N_previous_initializations(
```

Comment:
I  suggest changing the name to `get_dates_of_previous_initializations` as this way the function would follow [PEP8](https://peps.python.org/pep-0008/).